pkg/runtime/wfengine/workflow.go

```diff
@@ -556,6 +590,7 @@ func (wf *workflowActor) createReliableReminder(ctx context.Context, actorID str
 		return reminderName, fmt.Errorf("failed to encode data as JSON: %w", err)
 	}
 
+	// Do I have to add some sort of time instance here, which can be checked in "run-workflow" function for latency calculations ?
```

Comment:
Which latency calculation?

Reply:
I was wondering if the current method is correct for recording latency.